### PR TITLE
Add options param to createFormBuilders

### PR DESCRIPTION
**Pull request by dbollaer**
Here you go, I used the method option to check if my options were passed to the form.
I hope this will be enough. If not please provide additional feedback so I can improve these tests.

### CODE REVIEW

File: Tests/FormGeneratorTest.php [hunk 1]
```diff
@@ -58,6 +58,26 @@ public function testNamedForm()
         $this->assertSame('my_form', $form->getName());
     }
     
+
+    public function testNamedFormWithOptionsMethodGet(){
```
Comment (malarzm) [inline Tests/FormGeneratorTest.php]: I think that one test per method will be enough, they're checking the same thing anyway. Also please keep the coding style, `{` should go to new line, mind spaces around `(` and `,`

**Comment by dbollaer** [comment]
Thank you for your feedback. I have updated my tests. I will try to be more careful in the future.

**Comment by malarzm** [comment]
Thanks a bunch @dbollaer! I'll consider this as a bugfix to release quicker in 1.1.1, theoretically this could be seen as a shortcoming in our API :)

**Comment by malarzm** [comment]
@dbollaer I've just changed the milestone to 1.2.0, once I'm done with adding new Symfony's `autoconfigure` as an option I'll release new version (hopefully tomorrow).

**Comment by dbollaer** [comment]
Thank you for the update!

**Pull request merged**